dts/bindings/timer/telink,timer-gpio.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2024, Linaro Limited
```

Comment:
Why "Linaro Limited" instead of Telink ?
Check our other files headers

Reply:
Changed